apache/superset
# feat: hack db connection modal  ctabtns

### SUMMARY
This pr creates two cta buttons to create dataset or to go to sqllab in the db connection modal after a user connects their db

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
after

https://user-images.githubusercontent.com/17326228/167940966-59f70052-dd76-4303-b9aa-724088ea21d3.mov



### TESTING INSTRUCTIONS
Go to database connection modal and connect a db. There should be two cta button that take you to create dataset or to sql lab.

### ADDITIONAL INFORMATION
